release-notes/release-types.md

```diff
@@ -0,0 +1,32 @@
+# Release Types
+
+Each .NET release is defined as either **Long Term Support (LTS)** or **Current**. The difference is support time frame, as defined below:
+
+* **LTS** releases are supported for three years. They are intended for users that want the stability and lower cost of maintaining an application on a single (major.minor) .NET version for an extended period.
+* **Current** releases are supported for (typically) fifteen months. They are intended for users that want to take advantage of the newest features and improvements and to stay on the leading edge of .NET innovation. Current release users need to upgrade to later .NET releases more often to stay in support.
+
+LTS and Current releases have many similarities. The .NET team follows the same software engineering and release processes for both, including for security, compatibility, and reliability.
+
+Servicing policies are the same both release types. We use a single "bug bar" to decide if a change is warranted and safe for  servicing updates. In fact, a given fix is often applied to multiple [servicing branches](https://github.com/dotnet/core/blob/master/daily-builds.md#servicing-releases), independent of release type. Monthly patch releases are typical, including adding support for new operating system versions.
+
+## Release Support Policies
+
+.NET Core releases are supported according to the following policies.
```

Comment:
What do you think of adding descriptions of the maintenance periods. Here's the relevant section with refs emphasized from https://dotnet.microsoft.com/platform/support/policy/dotnet-core. Not suggesting the entire example be used, just including the context.

> How do the different support tracks work?

>Customers choosing LTS will need the latest patch update installed to qualify for support. If a system is running 3.1 and 3.1.x has been released, 3.1.x will need to be installed as a first step. Once a patch update has been installed, applications will begin using the update by default. LTS releases will be supported for 3 years after general availability, *or for a 12-month Maintenance period after the next LTS release ships, whichever is longer*.

>In addition to staying current with the latest patch update, customers using Current need to update as new minor versions are released to stay in support. The latest released minor version will become the minimum serviceable baseline after release. *After a 3-month Maintenance period, the previous minor version will no longer be supported*. For example, after 3.1 was released, systems running version 3.0 had 3 months to update to 3.1 to remain eligible for support.

Reply:
I was wanting to talk to you about the maintenance period concept. I get how it makes sense in theory and is evidence of being a mature organization, however, I still wonder if we need the extra concept. I'm looking to remove concepts that don't add significant value. Is there a significant change in our behavior or customer behavior that makes the maintenance concept warranted? I'd argue it is more warranted for the LTS release than the Current one. The Current maintenance period is so short such that it exactly aligns with its very last period of life. Like, if maintenance was just one month, it wouldn't make sense as a concept. I'd argue that one month and three months are the same. The 12 months of LTS maintenance is more interesting. I'd be much more compelled if maintenance was paid. It would make a more sense (to me) at that point. Because, today, we could just as easily say that the bar for functional fixes will be raised over time, and rises more after a new LTS is released. Thoughts?

Reply:
The idea is to have a clear definition of time where customers can make the move to the next version and still be eligible for critical updates. Back when we were first developing the lifecycles, we talked about this as a 'grace period' without writing it down in public. The 'maintenance' (name chosen as it's in use by Ubuntu and others to denote the after-mainline-support timeframe) support phase is meant to provide a clear public definition of the period for all release types. I agree the period is pretty short for Current releases but feel it important that lifecycle definitions be consistent across release types to mitigate the need for interpretation.

Reply:
Just to complete the conversation, I added a section on maintenance in response to your feedback. Are you happy with it?

Reply:
Yep, looks good. Thanks!